assets/src/edit-story/components/inspector/prepublish/prepublishChecklistProvider.js

```diff
@@ -47,46 +53,87 @@ function PrepublishChecklistProvider({ children }) {
     return { ...story, pages };
   });
 
+  const isStoryLoaded = story?.pages.length > 0;
+  const isChecklistEmpty = checkpointState === PPC_CHECKPOINT_STATE.NO_ISSUES;
+
   const [currentList, setCurrentList] = useState([]);
+  const [isChecklistReviewRequested, setIsChecklistReviewRequested] = useState(
+    false
+  );
 
   const handleRefreshList = useCallback(async () => {
     const pagesWithSize = story.pages.map((page) => ({
       ...page,
       pageSize,
     }));
 
-    // discern PPC types of messages based on checkpointState
```

Comment:
Not running checks was causing a lot of kooky side effects here for edge case, I was having to track a lot of things with refs to see if the checklist was valid or not. Normally it'd be fine but these are the cases where I was getting thrown: 

1. Story that is "perfect" according to checklist standards that is saved as a draft and then come back to (hit refresh):  the checklist status is completely abandoned and nothing's been loaded until I click on the checklist tab. If I just hit publish without going to that tab I'm going to get the dialog and that will send me to disabled tabs becasue no checks were present yet. 
2. I try to publish a blank story (just 1 page that has nothing on it) so nothing in the checklist has registered yet, I will get sent to the checklist and have a flash of "no errors!" state. 

Anyways, this way it's just a lot simpler. I am checking for checkpoints before rendering content of each checkpoint section instead.